web/crux-ui/locales/en/common.json

```diff
@@ -95,6 +95,7 @@
   "pipelines": "Pipelines",
   "storages": "Storages",
   "configBundles": "Config bundles",
```

Comment:
If you changed "config bundles" to singular, why are you leaving this as plural?

Reply:
Cause this is the translation for the side-menu, the other is the `routes.ts`.

Reply:
Okey, thanks for the clarification.